plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -540,7 +544,9 @@ export default createWidget("discourse-poll", {
     const showResults =
```

Comment:
This `showResults` variable should be kept in sync with the `can_see_results?(user)` method.